lib/utils/display_name_widget.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:fluffychat/presentation/model/presentation_contact.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+
+class BuildDisplayName extends StatelessWidget {
+  final ProfileInformation? profile;
```

Comment:
don't put the wholr ProfileInformation inside, this will make widget very hard to reuse

Reply:
Friendlyyyyyyyyy